org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/ProcessConsole.java

```diff
@@ -95,8 +96,7 @@ private void monitor(InputStream input, PublishSubject<String> subject) {
                     subject.onComplete();
                     return;
                 }
-                // TODO The buffer data may be not simple UTF8 chars,

```

Comment:
so user cannot specify the encoding but specified with UTF-8?

Reply:
on vscode side, it always parse the json protocol data with UTF-8, so we need keep the encoding format consistent.